src/node/service_map.h

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the Apache 2.0 License.
+#pragma once
+
+#include "crypto/pem.h"
+#include "kv/map.h"
+
+namespace ccf
+{
+  template <typename K, typename V>
+  using ServiceMap = kv::MapSerialisedWith<
```

Comment:
Should this just become the new `kv::Map`?

Reply:
No - we shouldn't change the schema for app tables like this (though maybe we deprecate `kv::Map` at some point if we have a few sensibly named alternatives we want them to pick from, to make the serialization choice more explicit). And keeping a separate `ServiceMap` name is very useful for distinguishing 'our' tables.